Gemfile

```diff
@@ -51,6 +51,7 @@ gem 'sidekiq'
 gem 'sidekiq-cron', '~> 1.1'
 gem 'skylight'
 gem 'validate_url', '~> 1.0.8'
+gem 'view_component'
```

Comment:
this gem has already been added in the school group overview PR from last week?

Reply:
That PR won the race to merge